profiles/apparmor/template.go

```diff
@@ -37,14 +37,5 @@ profile {{.Name}} flags=(attach_disconnected,mediate_deleted) {
   deny /sys/fs/cg[^r]*/** wklx,
   deny /sys/firmware/efi/efivars/** rwklx,
   deny /sys/kernel/security/** rwklx,
-
-{{if ge .Version 208095}}
-  # suppress ptrace denials when using 'docker ps' or using 'ps' inside a container
-  ptrace (trace,read) peer=docker-default,
```

Comment:
Haha, no problem. I'll need to re-add the version checks (the old version didn't set the version so this was never enabled).